### PR TITLE
va-checkbox and va-radio-option: Fix for Voice Control support

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://2952-voice-control-checks-and-radios--65a6e2ed2314f7b8f98609d8.chromatic.com

---

## Description
Closes [#2952](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2952)

Fixes Voice Control support by manually asserting the "checkbox" and "radio" roles on the va-checkbox and va-radio-option labels within the components. This is allowed by WCAG, but because it is not the "preferred" method - which states that semantic elements (in this case, the input elements) should be used wherever possible - I had to add some "ignore rule" statements to the AxeCheck tests.

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [X] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ describe('va-checkbox-group', () => {
     `,
     );
 
-    await axeCheck(page);
+    await axeCheck(page, ['aria-allowed-role']);
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts:25]: >  I had to add some "ignore rule" statements to the AxeCheck tests.

Do we think there might be any issues in vets-website tests where `cy.axeCheck();` is being called?

Comment (Andrew565) [reply packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts:25]: I don't know, but it's possible. Would the vets-website axe checks pierce the shadow dom?

**Review by rsmithadhoc — APPROVED** [review]
This looked good in my testing using Mac Voice Control. I'll bring this up at the next USWDS a11y meeting to see if there is a particular reason they went with that implementation. But this should be a good workaround for now, thanks!

File: packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts (hunk 1 above)
Comment (Andrew565) [reply packages/web-components/src/components/va-checkbox-group/test/va-checkbox-group.e2e.ts:25]: @jamigibbs I ran many of the vets-website Cypress tests with this update and didn't receive any axe check errors.

**Comment by jamigibbs** [comment]
Sorry @Andrew565 I didn't mean to merge this on your behalf. I meant to just approve ✅

**Comment by Andrew565** [comment]
Hehe, all good. Was ready to merge it anyways!

**Pull request merged**